packages/amazon-bedrock/src/internal/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from '../bedrock-chat-language-model';
+export * from '../bedrock-chat-settings';
```

Comment:
file / package not needed

Reply:
Sorry I'm not following, are you saying to remove this file completely, or just remove the bedrock-chat-settings export line?

Reply:
remove the internal export completely. I only have it for openai bc it's needed by azure openai. otherwise I try to keep the api surface as small as possible

Reply:
removed